web/samples_index/lib/src/carousel.dart

```diff
@@ -0,0 +1,210 @@
+// Copyright 2020 The Flutter team. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file
+
+import 'dart:html';
+
+class Carousel {
+  final bool withArrowKeyControl;
+
+  final container = querySelector('.slider-container');
+  var slides = querySelectorAll('.slider-single');
+
+  var lastSlide;
+  var currentSlide;
+  var prevSlide;
+  var nextSlide;
```

Comment:
nitpick: give these a type declaration

```suggestion
  Element lastSlide;
  Element currentSlide;
  Element prevSlide;
  Element nextSlide;
```